pkg/deployment/note/params.go

```diff
@@ -20,20 +20,17 @@ package note
 import (
 	"errors"
 
+	multierror "github.com/hashicorp/go-multierror"
+
 	"github.com/elastic/ecctl/pkg/deployment"
 )
 
 var (
-	errEmptyNoteMessage = errors.New("note message cannot be empty")
-	errEmptyUserID      = errors.New("user id cannot be empty")
-	errEmptyNoteID      = errors.New("note id cannot be empty")
+	errEmptyNoteMessage = "note message cannot be empty"
```

Comment:
Again I prefer consistency so if  we plan to keep `comment` as the term our user are seeing then I'd use the same term all over the place :)